pom.xml

```diff
@@ -13,7 +13,7 @@
     <groupId>org.mule.modules</groupId>
     <artifactId>mule-xml-module</artifactId>
     <packaging>mule-extension</packaging>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.0.1-SNAPSHOT</version>
```

Comment:
this version should point to the next major - 3.0.0-SNAPSHOT

Reply:
Sorry my mistake, we should keep 2.0.0. I was confused @zelisinho

Reply:
Fixed